test/lit.cfg

```diff
@@ -311,6 +311,40 @@ if test_sdk_overlay_dir is not None:
 test_options = os.environ.get('SWIFT_TEST_OPTIONS')
 config.swift_test_options = ' ' + test_options if test_options else ''
 
+def get_clang_library_dir():
+    llvm_lib_clang = os.path.join(config.llvm_libs_dir, 'clang')
+    if not os.path.exists(llvm_lib_clang):
+        return ''
+
+    versions = os.listdir(llvm_lib_clang)
+    if not len(versions):
+        return ''
+
+    version = max(versions)
+    return os.path.join(llvm_lib_clang, version, 'lib',
+                        platform.system().lower())
+
+config.clang_library_dir = get_clang_library_dir()
```

Comment:
Looks like it's `stdlib/public/SwiftShims/CMakeLists.txt`. It's specifically only handling headers right now.

I'm not sure what you mean about Xcode. I just generate a project using `utils/build-script --xcode -rt`. (Actually it's a more complicated preset, but that's the gist of it.)